Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/handler/AzureExceptionHandler.java

```diff
@@ -53,11 +56,13 @@ public static void onRxException(final Throwable e) {
     }
 
     public void handleException(Throwable throwable, @Nullable AzureExceptionAction... action) {
+        log.log(Level.WARNING, "caught an error in AzureExceptionHandler", throwable);
```

Comment:
I prefer to log.log(Level.WARNING, throwable);

Reply:
so do I, but there is no such method.